package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "irma-frontend-packages",
+  "license": "MIT",
+  "browsersList": "> 0.25%, not dead",
+  "devDependencies": {
+    "eslint": "^7.21.0",
+    "eslint-config-prettier": "^8.1.0",
+    "eslint-plugin-compat": "^3.9.0",
+    "eslint-plugin-prettier": "^3.3.1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/privacybydesign/irma-frontend-packages"
+  }
+}
```

Comment:
I now added a package.json to the root package to prevent dependency pollution. Most IDE's however don't pick-up eslint automatically due to this. Then I should add it as dependency in all `package.json`'s in the subdirectories. That is however also weird, because then a dev dependency is included there while the configuration for the dev package (`.eslintrc.json`) is not in that repository. That might be weird for build artifacts of the individual packages. It's a bit like choosing the lesser of two evils.